### ISSUE TITLE
[ENHANCEMENT] Karas Notation

### ISSUE THREAD

**Issue by zacharycayetano** [danielgarthur/neanes#425]
Karas/Extended notation would be a nice addition!

**Comment by cryptorick** [comment]
@danielgarthur I'm a user who is NOT in favor of Karas notation being added, but if you do go that route, would you please add a configuration item (like a checkbox) so I can turn it off. :smile: (I wouldn't want the pallette and toolbar to get larger or busier than they are already.)  BTW this software is the bomb! Thank you!

**Comment by nikostheotokatos** [comment]
> [@danielgarthur](https://github.com/danielgarthur) I'm a user who is NOT in favor of Karas notation being added, but if you do go that route, would you please add a configuration item (like a checkbox) so I can turn it off. 😄 (I wouldn't want the pallette and toolbar to get larger or busier than they are already.) BTW this software is the bomb! Thank you!
I totally agree: Karas notation is totally useless in today's byzantine music and not approved from Patriarchate Music Commission. If you want to add more notes, you simply write them. No need to write a "okseia" of "lygisma" or other old notation symbols in order to describe what notes the chanter will chant. If you truly want to add old notation byzantine music support, you have to add much more than Karas' notation neumes. But this is useful for old notation purpose research, which is respectful, but not for writing today's byzantine music, which, I suppose, is the purpose of this amazing software. It's preferable to spend your time to bug fixes or adding lucking features of existing notation, don't waste your time with issues that have been resolved more than 100  years before.

BTW, this software is by far the most user friendly byzantine music writer software. I have also tried other software and never managed to write music quickly. But this is the only one that allowed me to write byzantine music immediately after reading your simple and clear software guide. The interface is excellent, all functions are where you expect to be found! The programmer is a genius, your offer to the byzantine music community is so generous! Keep up the good work, THANK YOU!!!
